fides/templates/s3/s3-bucket.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Capabilities.APIVersions.Has "s3.services.k8s.aws/v1alpha1" -}}
+{{- with .Values.s3 -}}
+{{- if .createS3Bucket -}}
+apiVersion: s3.services.k8s.aws/v1alpha1
+kind: Bucket
```

Comment:
Wow, you really can define almost any resource in k8s. Hadn't encountered ACK before reviewing this PR.